appOPHD/UI/MiniMap.cpp

```diff
@@ -79,18 +79,18 @@ void MiniMap::draw() const
 		}
 	}
 
-	for (auto minePosition : mTileMap.oreDepositLocations())
+	for (auto oreDepositPosition : mTileMap.oreDepositLocations())
 	{
-		OreDeposit* mine = mTileMap.getTile({minePosition, 0}).oreDeposit();
-		if (!mine) { break; } // avoids potential race condition where a mine is destroyed during an updated cycle.
+		OreDeposit* oreDeposit = mTileMap.getTile({oreDepositPosition, 0}).oreDeposit();
+		if (!oreDeposit) { break; } // avoids potential race condition where an Ore Deposit is destroyed during an updated cycle.
```

Comment:
Unrelated to this PR, though "race condition" tends to imply a multithreading issue, and this game isn't multithreaded. We may want to revisit this comment later and perhaps update it for clarity over the actual problem it's trying to prevent.